wger/exercises/models.py

```diff
@@ -153,6 +153,24 @@ def delete(self, *args, **kwargs):
         super(ExerciseCategory, self).delete(*args, **kwargs)
 
 
+class Variation(models.Model):
+    """
+    Variation ids for exercises
+    """
+
+    def __str__(self):
+        """
+        Return a more human-readable representation
+        """
+        return self.name
```

Comment:
The variation doesn't have a name, better put something like return f'variation {self.id}' here